samples/matter/window_covering/src/app_task.cpp

```diff
@@ -261,3 +152,30 @@ void AppTask::ChipEventHandler(const ChipDeviceEvent *event, intptr_t /* arg */)
 		break;
 	}
 }
+
+CHIP_ERROR AppTask::Init()
+{
+	/* Initialize Matter stack */
+	ReturnErrorOnFailure(Nordic::Matter::InitServer(MatterEventHandler));
+
+	if (!GetBoard().Init(ButtonEventHandler)) {
+		LOG_ERR("User interface initialization failed.");
+		return CHIP_ERROR_INCORRECT_STATE;
+	}
+
+	WindowCovering::Instance().PositionLEDUpdate(WindowCovering::MoveType::LIFT);
```

Comment:
This runs the code that accesses the attribute store before the Matter data model is initialized. Even if it works, the behavior is undefined, so maybe this should be moved to `emberAfWindowCoveringClusterInitCallback`?

Reply:
Yes, or `mPostServerInitClbk` I guess.